### PR TITLE
softdevice_controller: Production support for Periodic Adv on nRF53

### PR DESCRIPTION
**Pull request by jori-nordic**
Mark Periodic Advertising as not experimental on nRF53.

Signed-off-by: Jonathan Rico <jonathan.rico@nordicsemi.no>

### CODE REVIEW

**Review by rugeGerritsen — APPROVED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Comment by b-gent** [comment]
Do we have a manifest update for this in sdk-nrf?

**Comment by b-gent** [comment]
@jori-nordic  please open a manifest update PR, we can't merge this unless the CI passes on it.

**Pull request merged**
